WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/ManagementNewsCardUseCase.kt

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases
+
+import kotlinx.coroutines.CoroutineDispatcher
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.store.StatsStore.ManagementType
+import org.wordpress.android.modules.UI_THREAD
+import org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase.StatelessUseCase
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.BigTitle
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.DialogButtons
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ImageItem
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.NavigationAction
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Tag
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Text
+import org.wordpress.android.ui.stats.refresh.utils.NewsCardHandler
+import org.wordpress.android.viewmodel.ResourceProvider
+import javax.inject.Inject
+import javax.inject.Named
+
+class ManagementNewsCardUseCase
+@Inject constructor(
+    @Named(UI_THREAD) private val mainDispatcher: CoroutineDispatcher,
+    private val resourceProvider: ResourceProvider,
+    private val newsCardHandler: NewsCardHandler
+) : StatelessUseCase<Boolean>(ManagementType.NEWS_CARD, mainDispatcher) {
+    override suspend fun loadCachedData() = true
+
+    override suspend fun fetchRemoteData(forced: Boolean): State<Boolean> = State.Data(true)
+
+    override fun buildLoadingItem(): List<BlockListItem> = listOf()
+
+    override fun buildUiModel(domainModel: Boolean): List<BlockListItem> {
+        val editText = resourceProvider.getString(R.string.stats_management_edit)
```

Comment:
Tiny nitpick - but maybe we can rename it to something like 'highlightedWord` to avoid confusion with `EditText`?

Reply:
makes sense